app/lib/bk/compat/parsers/circleci/orbs.rb

```diff
@@ -64,6 +61,54 @@ def install_commands(version, install_dir)
           end
         end
 
+        def translate_docker_build(config)
+          [
+            ("# Attaching workspace at #{config['attach-at']}" if config['attach-at']),
```

Comment:
What does this comment mean? Because I can not see the `attach-at` being used anywhere else